### PR TITLE
Revert "<chrono>: Cache QueryPerformanceFrequency() and divide just o…

### LINKED ISSUE #971: <chrono>: std::chrono::steady_clock::now() overflow
**Describe the bug**
I encountered an overflow issue with std::chrono::steady_clock::now() after ~923 seconds (~15 minutes).

Just copied some source from the original implementation to demonstrate the issue.

**Test sample**
```
#include <cassert>
#include <chrono>

_NODISCARD static long long _Scale_large_counter(const long long _Ctr, const long long _Freq) noexcept {
    const long long _Whole = (_Ctr / _Freq) * std::nano::den;
    const long long _Part = (_Ctr % _Freq) * std::nano::den / _Freq;
    return _Whole + _Part;
}

int main(int argc, char* argv[])
{
    static volatile long long _Cached_freq = LLONG_MAX;
    static volatile long long _Cached_ctr_base = LLONG_MAX;
    static volatile long long _Cached_result_base = LLONG_MAX;

    {
        const long long _Freq = _Query_perf_frequency();
        const long long _Ctr = _Query_perf_counter();
        const long long _Result = _Scale_large_counter(_Ctr, _Freq);

        _Cached_freq = _Freq;
        _Cached_ctr_base = _Ctr;
        _Cached_result_base = _Result;
    }

    const long long _Freq_from_cache = _Cached_freq;
    const long long _Ctr_base = _Cached_ctr_base;
    const long long _Result_base = _Cached_result_base;

    for (long long i = 1; i > 0; i += 1) {
        const long long _Ctr = _Ctr_base + i * 10000000;
        const long long now = _Result_base + (_Ctr - _Ctr_base) * std::nano::den / _Freq_from_cache;
        assert(now > _Result_base);
    }

    return 0;
}
```

**Expected behavior**
The overflow should happen at the earliest after 100 years as described on MSDN.
https://docs.microsoft.com/en-us/windows/win32/sysinfo/acquiring-high-resolution-time-stamps

```
How often does QPC roll over?

Not less than 100 years from the most recent system boot, and potentially longer based on the underlying hardware timer used. For most applications, rollover isn't a concern.
```

**STL version**
* Option 1: Visual Studio version
    ```
    Microsoft Visual Studio Enterprise 2019 Preview
    Version 16.7.0 Preview 3.0
    ```

**Additional context**
I think the formula to compute _now_ is wrong:
 _Result_base + (_Ctr - _Ctr_base) * std::nano::den / _Freq_from_cache

The multiplication by **std::nano::den** results in a negative value which then is divided.

The formula should be:
 _Result_base + (_Ctr - _Ctr_base) * **(std::nano::den / _Freq_from_cache)**

### PR DESCRIPTION
**Pull request by AlexGuteniev**
…nce (#653)"

This reverts commit ad1a26ad976b495ce1558afba7845c49d51fd2d5.

Fixes #971.

### CODE REVIEW

**Comment by BillyONeal** [comment]
Why do you want to revert this?

**Comment by BillyONeal** [comment]
Oh I see the linked issue

**Comment by cbezault** [comment]
Why not just fix the calculation?

**Comment by AlexGuteniev** [comment]
The proposed fix in #971 introduces precision loss if `den`  does not divide by frequency (observed, although rarely).

There are still ways to optimize. `_div128` is one of options. Another is to special case 10'000'000 frequency, as it is common. But I think the first thing to do is to fix the bug. And then probably don't touch this anymore.

File: stl/inc/chrono [hunk 1]
```diff
@@ -608,41 +607,8 @@ namespace chrono {
 #pragma warning(disable : 28112) // A variable which is accessed via an Interlocked function must
                                  // always be accessed via an Interlocked function.
```
Comment (StephanTLavavej) [inline stl/inc/chrono:608]: `#pragma warning push/disable/pop` was added for the `_InterlockedMeow` calls. This revert is removing the `pop` but not the `push/disable`.

**Comment by StephanTLavavej** [comment]
Thanks again for fixing this so quickly! :smiley_cat:

**Pull request merged**